src/Microsoft.Framework.ConfigurationModel/ConfigurationFocus.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Framework.ConfigurationModel
 {
-    public class ConfigurationFocus : IConfiguration
+    internal class ConfigurationFocus : IConfiguration
```

Comment:
https://github.com/aspnet/Configuration/issues/142 said to move this to a `.Internal` sub-namespace, not to make the type `internal`.